avro-builder/src/test/java/com/linkedin/avroutil1/builder/SchemaBuilderTest.java

```diff
@@ -97,6 +97,7 @@ private File locateTestProjectsRoot() {
     if (!file.isDirectory() || !file.canRead()) {
       throw new IllegalStateException("test-projects root folder " + file.getAbsolutePath() + " not a folder or is unreadable");
     }
+    System.err.println(file.getAbsolutePath());
```

Comment:
Is this added for debugging purposes? 
If yes, Add a prefix like "File absolute path:" for context and move from err.println -> out.println if its just info

Reply:
Yes, it was for debugging purposes. I will remove